mono/mini/method-to-ir.c

```diff
@@ -1740,11 +1740,14 @@ emit_push_lmf (MonoCompile *cfg)
 	/*
 	 * Emit IR to push the LMF:
 	 * lmf_addr = <lmf_addr from tls>
-	 * lmf->lmf_addr = lmf_addr
-	 * lmf->prev_lmf = *lmf_addr
-	 * *lmf_addr = lmf
+	 * if (lmf_addr != NULL) {
+	 *   lmf->lmf_addr = lmf_addr
```

Comment:
Why is this needed ? The lmf should only be null if the thread is not attached to the JIT.

Reply:
the `interp_in` wrapper is called _before_ the `native-to-managed` wrapper which would to do the attaching. if it's called from an unattached thread, we don't care about the LMF in this wrapper; in other scenarios we might do.

Reply:
Not sure why its called before, but if that really happens, then the null check should be emitted only in interp_in wrappers, not everywhere.